README.md

```diff
@@ -75,10 +85,12 @@ Create a `secretproviderclasses` resource to provide provider-specific parameter
 
     ```
 
-    | Name           | Required | Description                                                     | Default Value |
-    | -------------- | -------- | --------------------------------------------------------------- | ------------- |
-    | provider       | yes      | specify name of the provider                                    | ""            |
-    | usePodIdentity | no       | specify access mode: service principal or pod identity          | "false"       |
+    | Name                   | Required | Description                                                     | Default Value |
+    | --------------         | -------- | --------------------------------------------------------------- | ------------- |
+    | provider               | yes      | specify name of the provider                                    | ""            |
+    | usePodIdentity         | no       | specify access mode: service principal or pod identity          | "false"       |
+    | useVMManagedIdentity   | no       | specify access mode: use a service principal or pod identity or vm managed identity    |  "false"|
```

Comment:
same here - `OPTIONAL available for version > 0.0.4` in the Required column for both `useVMManagedIdentity` and `userAssignedIdentityID` (similar to how we specify for `objectAlias` property)